<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToolClassClass</class>
 <widget class="QMainWindow" name="ImageToolClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageToolClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(12, 34, 56);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,20">
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,2,10,8,20,1">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,10">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../MatchTest/Resources/Quectel LOGO-20220215.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>638</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_28" stretch="1,1,1,1,0">
                <property name="spacing">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QToolButton" name="toolButton_4">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>11</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>设置</string>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextOnly</enum>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_5">
                  <property name="font">
                   <font>
                    <family>Times New Roman</family>
                    <pointsize>11</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>日志</string>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextOnly</enum>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_6">
                  <property name="font">
                   <font>
                    <family>Liberation Serif</family>
                    <pointsize>11</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>语言</string>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextOnly</enum>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_7">
                  <property name="font">
                   <font>
                    <family>Liberation Serif</family>
                    <pointsize>11</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>帮助</string>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextOnly</enum>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_title">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>678</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,11,0">
                <item>
                 <widget class="QToolButton" name="toolButton">
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../../MatchTest/Resources/窗口最小化.png</normaloff>../../../MatchTest/Resources/窗口最小化.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_2">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../../MatchTest/Resources/expand-全屏化.png</normaloff>../../../MatchTest/Resources/expand-全屏化.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_3">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../../MatchTest/Resources/关闭.png</normaloff>../../../MatchTest/Resources/关闭.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,15,1">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="toolButton_provious">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../../MatchTest/Resources/切换器左.png</normaloff>../../../MatchTest/Resources/切换器左.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_image">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_next">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../../MatchTest/Resources/切换器右.png</normaloff>../../../MatchTest/Resources/切换器右.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="tabPosition">
              <enum>QTabWidget::North</enum>
             </property>
             <property name="tabShape">
              <enum>QTabWidget::Rounded</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="movable">
              <bool>false</bool>
             </property>
             <property name="tabBarAutoHide">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>参数设置</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_5">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <property name="spacing">
                <number>2</number>
               </property>
               <item row="0" column="0">
                <widget class="QFrame" name="frame_2">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <property name="leftMargin">
                   <number>2</number>
                  </property>
                  <property name="topMargin">
                   <number>2</number>
                  </property>
                  <property name="rightMargin">
                   <number>2</number>
                  </property>
                  <property name="bottomMargin">
                   <number>2</number>
                  </property>
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,1,1,1,1">
                      <item>
                       <widget class="QLabel" name="label_imageNumber">
                        <property name="font">
                         <font>
                          <family>微软雅黑</family>
                          <pointsize>11</pointsize>
                          <weight>75</weight>
                          <bold>true</bold>
                         </font>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">color: rgb(238, 238, 236);</string>
                        </property>
                        <property name="text">
                         <string>图像源(0/0)</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QRadioButton" name="radioButton_switchImage">
                        <property name="font">
                         <font>
                          <family>微软雅黑</family>
                          <pointsize>11</pointsize>
                          <weight>50</weight>
                          <bold>false</bold>
                         </font>
                        </property>
                        <property name="text">
                         <string>自动切换</string>
                        </property>
                        <property name="checked">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btn_openImage">
                        <property name="styleSheet">
                         <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(27, 46, 71);</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>../../../MatchTest/Resources/历史图像.png</normaloff>../../../MatchTest/Resources/历史图像.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btn_imagefiles">
                        <property name="styleSheet">
                         <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(27, 46, 71);</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>../../../MatchTest/Resources/打开.png</normaloff>../../../MatchTest/Resources/打开.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btn_deleteImage">
                        <property name="styleSheet">
                         <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(27, 46, 71);</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>../../../MatchTest/Resources/删除.png</normaloff>../../../MatchTest/Resources/删除.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QListWidget" name="listWidget">
                      <property name="verticalScrollBarPolicy">
                       <enum>Qt::ScrollBarAlwaysOff</enum>
                      </property>
                      <property name="horizontalScrollBarPolicy">
                       <enum>Qt::ScrollBarAsNeeded</enum>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>210</width>
                        <height>175</height>
                       </size>
                      </property>
                      <property name="horizontalScrollMode">
                       <enum>QAbstractItemView::ScrollPerPixel</enum>
                      </property>
                      <property name="movement">
                       <enum>QListView::Static</enum>
                      </property>
                      <property name="flow">
                       <enum>QListView::LeftToRight</enum>
                      </property>
                      <property name="resizeMode">
                       <enum>QListView::Adjust</enum>
                      </property>
                      <property name="spacing">
                       <number>8</number>
                      </property>
                      <property name="viewMode">
                       <enum>QListView::ListMode</enum>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
